src/share/util/shr_const_mod.F90

```diff
@@ -7,81 +7,94 @@ MODULE shr_const_mod
 
    use shr_kind_mod
 
-   integer(SHR_KIND_IN),parameter,private :: R8 = SHR_KIND_R8 ! rename for local readability only
+   !----------------------------------------------------------------------------
```

Comment:
These changes are not okay, new variables need to follow naming convention and most lines in this file should not change from original - I will submit an update.